.github/workflows/check-release-notes.yml

```diff
@@ -0,0 +1,14 @@
+name: Check release notes
+
+on:
+  pull_request:
+    branches: '*'
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Check release notes
+        uses: actualbudget/actions/release-notes/check@main
```

Comment:
Not sure how possible it is, but it would be super cool if a comment was posted to the PR if this check fails with instructions what to do. Otherwise I see people will get confused as to what is this thing.

Reply:
When the check fails people will be able to click through to see the check failure (which will look a bit like this: https://github.com/actualbudget/actual/actions/runs/4400698422/jobs/7706297548). That said I will make a PR template that reminds people to fill it out.